workload/docs/deploy-baseline.md

```diff
@@ -11,7 +11,7 @@
   - **Environment** – Deployment Environment type (Development/Test/Production), will be used for naming and tagging purposes.
 - **Identity provider** blade
   - **Identity Service Provider** - Identity service provider (AD DS, AAD DS, AAD) that already exists and will be used for Azure Virtual Desktop.
-    - Microsoft Entra ID (AAD).
+    - Microsoft Entra ID.
     - Active Directory (AD DS).
     - Microsoft Entra ID Domain Services (AAD DS).
```

Comment:
AAD DS service hasn't been renamed to the new Microsoft Entra ID model, for now let's keep Azure AD DS (AAD DS)

Reply:
Done